public/video-ui/src/components/VideoAssets/VideoAssets.js

```diff
@@ -110,14 +110,14 @@ class VideoAssets extends React.Component {
   renderHeader() {
     const buttons = <div className="video-assets__buttons">
       <a className="button" onClick={this.showAssetForm}>
-        <Icon className="icon__edit" icon="add"/>
+        <Icon className="icon__edit icon__spacing" icon="add"/>
```

Comment:
`icon__spacing` is adding a `margin-right` but there's no content to the right of this `Icon`. Would be better to add the spacing to the `a` above instead.